packages/common/src/ceramic-api.ts

```diff
@@ -200,5 +200,5 @@ export interface MultiQuery {
   /**
    * Load a previous version of the stream based on unix timestamp
    */
-  atTime?: number
```

Comment:
We can't remove the existing atTime parameter or we'll break all existing applications that are currently using it.  We need to make this change in a backwards-compatibility-preserving way.  So we should continue to support the existing atTime field, while adding support for LoadOpts.  We should, however, mark the old atTime field as deprecated by adding `@deprecated` to the doc comment here, with a note to use `opts` instead.